pom.xml

```diff
@@ -804,13 +798,15 @@
     <module>distribution</module>
     <module>math-scala</module>
     <module>spark</module>
-    <module>spark-shell</module>
```

Comment:
Converting spark-shell is a nontrivial task that I don't have time to tackle. Spark itself has distinct shell code for 2.11 & 2.10. This change just formalizes excluding spark-shell from the 2.11 build; in a perfect universe the majority of this change wouldn't be needed.